### PR TITLE
Updated links in all three documents.

### PR DESCRIPTION
**Pull request by jrmerwin**

### CODE REVIEW

File: android-DL4JImageRecognitionDemo.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Contents
...
+This example application uses a neural network trained on the standard MNIST dataset of 28x28 greyscale 0..255 pixel value images of hand drawn numbers 0..9. The application user interace allows the user to draw a number on the device screen which is then tested against the trained network. The output displays the most probable numeric values and the probability score. This tutorial will cover the use of a trained neural network in an Android Application, the handling of user generated images, and the output of the results to the UI from a background thread. For a detailed guide demonstrating how to train and save the neural networks used in this application, please see this DL4J quickstart [tutorial](https://deeplearning4j.org/quickstart). More information on general prerequisites for building DL4J Android Applications can be found [here](https://github.com/deeplearning4j/deeplearning4j-docs/blob/gh-pages/android-prerequisites-configuration.md).
```
Comment (saudet) [inline android-DL4JImageRecognitionDemo.md]: The URL here is wrong. It points to the repository and opens up an editor.

File: android.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ layout: default
 <ul>
   <li>An Android device or emulator that runs API level 21 or higher, and has about 200 MB of internal storage space free. I strongly suggest you use an emulator first because you can quickly tweak it in case you run out of memory or storage space.</li>
   <li>Android Studio 2.2 or newer</li>
-  <li>A more in-depth look at using DL4J in Android Applications can be found <a href="https://deeplearning4j.org/android-Prereqs%20and%20Configuration" target="_blank" rel="nofollow">here.</a> This guide covers dependencies, memory management, saving device-trained models, and loading pre-trained models in the application.</li>
+  <li>A more in-depth look at using DL4J in Android Applications can be found <a href="https://github.com/deeplearning4j/deeplearning4j-docs/blob/gh-pages/android-prerequisites-configuration.md" target="_blank" rel="nofollow">here.</a> This guide covers dependencies, memory management, saving device-trained models, and loading pre-trained models in the application.</li>
```
Comment (saudet) [inline android.md]: This one too.

File: android-DL4JIrisClassifierDemo.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ layout: default
...
+The example application trains a small neural network on the device using Anderson’s Iris data set for iris flower type classification. For a more indepth look at optimizing android for DL4J, please see the Prerequisites and Configuration documentation [here](https://github.com/deeplearning4j/deeplearning4j-docs/blob/gh-pages/android-prerequisites-configuration.md). This application has a simple UI to take measurements of petal length, petal width, sepal length, and sepal width from the user and returns the probability that the measurements belong to one of three types of Iris (*Iris serosa*, *Iris versicolor*, and *Iris virginica*). A data set includes 150 measurement values (50 for each iris type) and training the model takes anywhere from 5-20 seconds, depending on the device.
```
Comment (saudet) [inline android-DL4JIrisClassifierDemo.md]: This one as well.

**Comment by crockpotveggies** [comment]
@jrmerwin can you open up a new PR to quickly fix these?

**Comment by jrmerwin** [comment]
yes I'll do it now

**Comment by jrmerwin** [comment]
@crockpotveggies ready to merge

**Pull request merged**